pkg/logs/wait_for_image.go

```diff
@@ -56,7 +56,11 @@ func (w *imageWaiter) resultOfImageWait(ctx context.Context, writer io.Writer, g
 		return w.waitBuild(ctx, writer, image.Namespace, image.Status.LatestBuildRef)
 	}
 
-	if image.Status.GetCondition(corev1alpha1.ConditionReady).IsFalse() {
+	if condition := image.Status.GetCondition(corev1alpha1.ConditionReady); condition.IsFalse() {
+		if condition.Message != "" {
+			return "", errors.Wrap(errors.New(condition.Message), fmt.Sprintf("update to image %s failed", image.Name))
```

Comment:
Could we use a single `errors.Errorf` here?

Reply:
ya I can change that